plugins/e57/io/Utils.cpp

```diff
@@ -215,15 +219,10 @@ std::pair<uint64_t, uint64_t> getPdalBounds(pdal::Dimension::Id id)
     switch (id)
     {
         case Dim::Red:
-            return {std::numeric_limits<uint16_t>::min(),
-                    std::numeric_limits<uint16_t>::max()};
         case Dim::Blue:
-            return {std::numeric_limits<uint16_t>::min(),
-                    std::numeric_limits<uint16_t>::max()};
         case Dim::Green:
-            return {std::numeric_limits<uint16_t>::min(),
-                    std::numeric_limits<uint16_t>::max()};
         case Dim::Intensity:
+        case Dim::Classification:
             return {std::numeric_limits<uint16_t>::min(),
```

Comment:
Pdal bounds for classification are uint8_t

Reply:
I don't think classification will float in any case.

Reply:
Done